### DISCUSSION TITLE
Build command for 'usedapp_io' application on vercel

### DISCUSSION THREAD

**Discussion by RohitKS7** [vercel/vercel#8968] [Help]
Hey, I'm trying to deploy my website (build on 'useDapp.io' compound react framework) on Vercel but this commands for project building is not working ( Vercel UI)

1. `yarn react-app:build`
2. `yarn build`
3. `yarn workspace @my-app/react-app build`

After that I tried to deploy with Vercel CLI

The problem I'm facing is, I'm confused!
which directory to use "as output", "as root" and should I use 'create-react-app' or others as framework while deploying since it consists contract and more node_module outside of react-app folder which is also important for the Application

> My Repo Link - https://github.com/RohitKS7/Uniswap_2.0

Can anyone guide me to what is the right path to deploy it.

**Comment by swarnava** [comment]
How are you building the project currently?

**Reply by RohitKS7** [reply]
From most outer package.json scripts not from react-app (inside packages). While building in local machine it all fine but while deploying it's not working.

**Reply by amyegan** [reply]
There is a guide for "[Deploying a Monorepo Using Yarn Workspaces to Vercel](https://vercel.com/guides/deploying-yarn-monorepos-to-vercel)" that might be what you need. It walks through the [monorepo](https://vercel.com/docs/concepts/monorepos#) configuration steps for Vercel deployment.
